mxcube3/ui/components/SampleQueue/HistorySampleItem.js

```diff
@@ -6,17 +6,17 @@ import './app.less';
 export default class HistorySampleItem extends React.Component {
   constructor(props) {
     super(props);
-    this.collapseSample = props.collapseSample.bind(this, props.id);
+    this.collapseSample = props.collapseSample.bind(this, props.data.clientID);
```

Comment:
I'm really sorry about that it seems like I missed a commit. Ill see if I can find it and add it :)